src/main.zig

```diff
@@ -2,11 +2,20 @@
 //! you are building an executable. If you are making a library, the convention
 //! is to delete this file and start with root.zig instead.
 const std = @import("std");
+const root = @import("root.zig");
```

Comment:
Right now, we actually don't use main, we don't need change this file. Since we develop a library not a application.